src/ripple/app/ledger/LedgerMaster.h

```diff
@@ -323,10 +324,10 @@ class LedgerMaster
     bool    mPathFindNewRequest;
 
     std::atomic <std::uint32_t> mPubLedgerClose;
-    std::atomic <std::uint32_t> mPubLedgerSeq;
+    std::atomic <LedgerIndex> mPubLedgerSeq;
     std::atomic <std::uint32_t> mValidLedgerSign;
-    std::atomic <std::uint32_t> mValidLedgerSeq;
-    std::atomic <std::uint32_t> mBuildingLedgerSeq;
+    std::atomic <LedgerIndex> mValidLedgerSeq {0};
```

Comment:
It looks like this is initialized in the constructor already (https://github.com/ripple/rippled/blob/develop/src/ripple/app/ledger/impl/LedgerMaster.cpp#L83).  I actually prefer the inline initialization you have here.  Consider treating the other atomics the same way.